From 65d30296fa643fdb376ff4c3a6c98763a31fa0bf Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Fri, 9 Jul 2021 15:09:09 +0200
Subject: [PATCH 1/1] Solus tweaks to default landingpage

- Set SolusDark as default Darktheme
- Set SolusLight as default Lighttheme
---
 kcms/landingpage/landingpage_kdeglobalssettings.kcfg | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/kcms/landingpage/landingpage_kdeglobalssettings.kcfg b/kcms/landingpage/landingpage_kdeglobalssettings.kcfg
index 6bcce1e..8765892 100644
--- a/kcms/landingpage/landingpage_kdeglobalssettings.kcfg
+++ b/kcms/landingpage/landingpage_kdeglobalssettings.kcfg
@@ -7,7 +7,7 @@
   <group name="General">
     <entry name="colorScheme" key="ColorScheme" type="String">
       <label>Color scheme name</label>
-      <default>BreezeLight</default>
+      <default>SolusDark</default>
     </entry>
   </group>
   <group name="KDE">
@@ -17,15 +17,15 @@
     </entry>
     <entry name="lookAndFeelPackage" key="LookAndFeelPackage" type="String">
       <label>Global Look and Feel package</label>
-      <default>org.kde.breeze.desktop</default>
+      <default>solusdark.desktop</default>
     </entry>
     <entry name="defaultLightLookAndFeel" key="DefaultLightLookAndFeel" type="String">
       <label>Global Look and Feel package, alternate</label>
-      <default>org.kde.breeze.desktop</default>
+      <default>soluslight.desktop</default>
     </entry>
     <entry name="defaultDarkLookAndFeel" key="DefaultDarkLookAndFeel" type="String">
       <label>Global Look and Feel package, alternate</label>
-      <default>org.kde.breezedark.desktop</default>
+      <default>solusdark.desktop</default>
     </entry>
     <entry name="animationDurationFactor" key="AnimationDurationFactor" type="Double">
       <label>Animation speed</label>
-- 
2.32.0

